docs/network.md

```diff
@@ -0,0 +1,16 @@
+# Network ports
```

Comment:
What if we make the section name not too technical? I have something in mind like Administrative requirements, Percona Platform administration details, etc